src/common/logger.rs

```diff
@@ -63,7 +63,7 @@ pub fn init_logger(role: LogRole) {
         .with_target("h2", Level::WARN)
         .with_target("tower", Level::WARN)
         .with_target("datenlord::async_fuse::fuse", Level::INFO)
-        .with_target("", Level::DEBUG);
+        .with_target("", Level::INFO);
```

Comment:
Would it be better to set `DEBUG` level in debug mode while `INFO` level in release mode?

Reply:
I think debug mode should be enabled when a bug is discovered and needs debugging. By default, I prefer info because INFO logs are concise and efficient.

Reply:
So when do we enable DEBUG level? Only in debugging? And `debug!` should never exist in formal codes?

Reply:
It's acceptable to have a certain amount of `!debug` logs, but only a small quantity. Compared to `!debug`, I'd prefer to see more `debug_assert!()` in the project. Currently, even running a simple load might generate 200,000 lines of info logs.

Reply:
> It's acceptable to have a certain amount of `!debug` logs, but only a small quantity. Compared to `!debug`, I'd prefer to see more `debug_assert!()` in the project. Currently, even running a simple load might generate 200,000 lines of info logs.

Make sense. Then we have more reasons to check the unneccessary `debug!` in metadata, remove them or replace then with `info!`. Maybe it's not needed to be done in this PR, but we should take cares of them.